GitVersion.yml

```diff
@@ -1,10 +1,18 @@
 mode: ContinuousDelivery
-next-version: 0.0.1
+next-version: 2.13.0
 branches:
+  feature:
+    mode: ContinuousDeployment
+    regex: ^feat[/-]
+    source-branches: ["develop", "master"]
```

Comment:
You have "main" in the .travis.yml file too?
Edit: also missing below x2

Reply:
Thanks for the review! Yeah, this was a copy and paste from enterprise-js and it slipped through. 

I'm not familiar with Semantic Release but I will check it out :)